packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -59,6 +63,14 @@ export interface IColumnHeaderCellProps extends IColumnNameProps, IProps {
     isColumnSelected?: boolean;
 
     /**
+     * If true, the column `name` will be replaced with a fixed-height skeleton and the
+     * `resizeHandle` will not be rendered. If passing in additional children to this component, you
+     * will also want to conditionally apply the `.pt-skeleton` class where appropriate.
+     * @default false
+     */
+    loading?: boolean;
```

Comment:
We picked `loading` earlier, we should stick with it everywhere. Otherwise, file an issue to change all instances of `loading` to `isLoading`. What matters here is consistency in naming